dandelion-voting/app/src/components/BlockTimerHelper.js

```diff
@@ -1,16 +1,14 @@
 import React from 'react'
 import { Help } from '@aragon/ui'
-import useBlockNumber from '../hooks/useBlockNumber'
+import { useBlockTime, useBlockLatest } from '../hooks/useBlock'
+import { getVoteRemainingBlocks } from '../vote-utils'
 
-const BlockTimerHelper = ({ vote, blockTime }) => {
-  const { data } = vote
-  const { endBlock, pending, startBlock, delayed, executionBlock } = data
-  const currentBlockNumber = useBlockNumber()
-  const remainingBlocks = pending
-    ? startBlock - currentBlockNumber
-    : delayed
-    ? executionBlock - currentBlockNumber
-    : endBlock - currentBlockNumber
+const BlockTimerHelper = ({ vote }) => {
+  const { upcoming, delayed } = vote.data
+  const { number: currentBlockNumber } = useBlockLatest()
```

Comment:
Where are we getting that currentBlockNumber from?

Reply:
That's the js syntax for getting the number property of the block and using the `currentBlockNumber` as the variable that will have the value.

Reply:
> That's the js syntax for getting the number property of the block and using the `currentBlockNumber` as the variable that will have the value.

cool! didn't know that